--- rubocop-0.18.1/lib/rubocop/config_loader.rb~	2014-02-17 09:12:45.000000000 +0200
+++ rubocop-0.18.1/lib/rubocop/config_loader.rb	2014-02-17 09:51:48.673259129 +0200
@@ -11,7 +11,7 @@
   # directories are inspected.
   class ConfigLoader
     DOTFILE = '.rubocop.yml'
-    RUBOCOP_HOME = File.realpath(File.join(File.dirname(__FILE__), '..', '..'))
+    RUBOCOP_HOME = File.realpath(File.dirname(__FILE__))
     DEFAULT_FILE = File.join(RUBOCOP_HOME, 'config', 'default.yml')
     AUTO_GENERATED_FILE = 'rubocop-todo.yml'
 
